.golden/swiftMultipleTypeVariableSpec/golden

```diff
@@ -1,4 +1,4 @@
-struct Data<A, B>: CaseIterable, Hashable, Codable {
+struct Data<A: Codable, B: Codable>: CaseIterable, Hashable, Codable {
```

Comment:
These need to be Hashable/Codable too, or this won't compile. Same for the LTree example.

Reply:
Thanks, I'll add this.

Reply:
To be clear, they need all the same conformance as the enclosing type in order for the enclosing type to conform. Except for `CaseIterable` which shouldn't even be applied to structs lol